src/io/aviso/taxi_toolkit/ui.clj

```diff
@@ -84,9 +84,8 @@
 
   [& el-val-or-entries]
   (let [el-val (if (= (count el-val-or-entries) 1)
-                 el-val-or-entries
+                 (first el-val-or-entries)
```

Comment:
Do we need that? `el-val` is then passed to `doseq`, so it should be a collection, shouldn't it?